Makefile

```diff
@@ -1,18 +1,34 @@
 MAKEFLAGS += --silent
 
-default:
-	echo No default target
+BUILD := $(shell git describe --dirty)
+LDFLAGS=-ldflags "-X github.com/liatrio/gh-trusted-builds-attestations/build.Version=$(BUILD)"
```

Comment:
Should be possible to use [`debug.ReadBuildInfo`](https://pkg.go.dev/runtime/debug#ReadBuildInfo) to get the Git information. IIRC `buildInfo.Main.Version` should return the Git tag or `devel` if there isn't one.

Reply:
I can see the commit sha in the build settings list, but not a tag unfortunately. 

```
vcs=git
vcs.revision=42dcfd9b42929b65ab0f93910ed2b42b3f1aff50
vcs.time=2023-05-24T16:05:07Z
vcs.modified=true
```